### DISCUSSION TITLE
Is there a chance I can get Student Developer Pack in my situation?

### DISCUSSION THREAD

**Discussion by WordsetterFak** [community/community#134444] [GitHub Education]
### Hi everyone,

I am currently enrolled in a university, recognized by github, which requires an academic email from the department of Computer Science to be registered in github, for the Student pro pack to be granted. Not only do I not have any academic id, I am enrolled in the department of Mathematics, however I am a developer as well and have a general interest towards Computer Science, having taken such courses. Is there a chance I can get github developer pro in my situation?

**Comment by Retro788** [comment]
It is really difficult, since they use the institutional email to verify your account and that is where you receive the confirmation emails for the academic program

**Comment by Retro788** [comment]
The other way you can do it is by contacting GitHub support, sending a certificate of studies and a photo of your student ID along with its expiration date and they can directly link you to the platform with your regular email, the response time to the email will take from 2 hours to 3 days, as I recall.

**Reply by WordsetterFak** [reply]
Thank you for your answer! Could you tell me, where I would send such an email, I have attempted to contact github support directly, but when I selected the github education category, I couldn't find a relevant reason, that was close to my request, from their dropdown.
